src/common/nullpo.c

```diff
@@ -32,6 +41,13 @@ void assert_report(const char *file, int line, const char *func, const char *tar
 
 	ShowError("--- %s --------------------------------------------\n", title);
 	ShowError("%s:%d: '%s' in function `%s'\n", file, line, targetname, func);
+#ifdef __GNUC__
+	size = backtrace (array, 10);
+	strings = backtrace_symbols (array, size);
+	for (i = 0; i < size; i++)
+		ShowError("%s\n", strings[i]);
+	free (strings);
```

Comment:
From man (http://linux.die.net/man/3/backtrace_symbols)
It need only #include <execinfo.h>